src/vcpkg/sourceparagraph.cpp

```diff
@@ -1339,6 +1339,8 @@ namespace vcpkg
         {
             ret.feature_paragraphs.push_back(std::make_unique<FeatureParagraph>(*feat_ptr));
         }
+
+        ret.extra_features_info = extra_features_info;
```

Comment:
I really wish this thing was just copyable, but it has this clone function and I'm being consistent with it.